pkg/abstractions/image/build.go

```diff
@@ -201,6 +263,8 @@ func (b *Builder) Build(ctx context.Context, opts *BuildOpts, outputChan chan co
 		ImageId:          baseImageId,
 		SourceImage:      &sourceImage,
 		SourceImageCreds: opts.BaseImageCreds,
+		Dockerfile:       dockerfile,
+		BuildCtxObject:   &opts.BuildCtxObject,
```

Comment:
I really don't like that this is top level on the ContainerRequest. It might seem out of scope but I think we should nest these under a BuildOptions struct on the container request

Reply:
Bumped it out